plugins/catalog/src/components/CatalogPage/DefaultCatalogPage.tsx

```diff
@@ -71,15 +72,14 @@ export function DefaultCatalogPage(props: DefaultCatalogPageProps) {
   const orgName =
     useApi(configApiRef).getOptionalString('organization.name') ?? 'Backstage';
   const createComponentLink = useRouteRef(createComponentRouteRef);
-
-  const { createButtonTitle } = useCatalogPluginOptions();
+  const { t } = useTranslationRef(catalogTranslationRef);
```

Comment:
not related to this PR but have we named `useTranslationRef` instead of `useTranslation` to avoid confusion between the hook exposed by backstage and the hook exposed by other libraries such as react-i18next?

Reply:
Yep that naming is to avoid confusion